src/components/EditCoursePage/EditCourseForm.jsx

```diff
@@ -307,6 +308,7 @@ export class BaseEditCourseForm extends React.Component {
                       </p>
                     </div>
                   }
+                  extraText="Image must be 1134x675 pixels in size."
```

Comment:
This does feel like we can pull this out of the tooltip now? Or did Carol want it in both places?

Reply:
Good question! Checked with Carol and she said both places